api/exec/update_env.go

```
@@ -13,15 +18,36 @@ func UpdateEnv(workspaceId string, applicationId string, key string, value strin
 	}
 	header["Authorization"] = "Bearer " + accessToken
 
-	body := map[string]string{}
-	body["newValue"] = value
-	envReq := envRequest{EnvList: body}
-	requestJson, err := json.Marshal(envReq)
+	updateEnvReq := updateEnvRequest{NewValue: value}
+	requestJson, err := json.Marshal(updateEnvReq)
```

golangci-lint

ineffectual assignment to err

(ineffassign)